src/components/ItemDetail/ItemDetail.js

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import Cite from './Cite';
+import ItemDetailMetadata from './ItemDetailMetadata';
+import FindThisItem from './FindThisItem';
+
+const ItemDetail = props => {
+  return (
+    <section className="item-section contain-970 item-categories-wrapper">
+      <div id="tab-container">
+        <ul id="tabs" role="tablist">
+          <li role="presentation">
+            <a
+              aria-controls="tab-panel1"
+              href="#tab-panel1"
+              id="tab1"
+              role="tab"
+            >
+              About this Item
+            </a>
+          </li>
+          <li role="presentation">
+            <a
+              aria-controls="tab-panel2"
+              href="#tab-panel2"
+              id="tab2"
+              role="tab"
+            >
+              Find this Item
+            </a>
+          </li>
+          <li role="presentation">
+            <a
+              aria-controls="tab-panel3"
+              href="#tab-panel3"
+              id="tab3"
+              role="tab"
+            >
+              Cite this Item
+            </a>
+          </li>
+        </ul>
+        <div id="tab-content">
+          <div aria-labelledby="tab-item-data" id="tab-panel1" role="tabpanel">
+            <ItemDetailMetadata item={props.item} />
+          </div>
+          <div aria-labelledby="tab-find-item" id="tab-panel2" role="tabpanel">
+            <FindThisItem item={props.item} />
+          </div>
+          <div aria-labelledby="tab-cite" id="tab-panel3" role="tabpanel">
+            <Cite item={props.item} />
+          </div>
```

Comment:
Could `<ItemDetailMetadata />`, `< FindThisItem />`, and `<Cite />` be a common component?   Is the UI layout the same within each?   I can't remember.  But if so, then a good pattern would be to calculate what line entries you want to pass into each, as an array, then inject that array into the possible common component.  Something like:
```
<SomethingTab items={arrayOfObjects} />
```